### ISSUE TITLE
Lag Llama dates in fine tuning and forecasting.

### ISSUE THREAD

**Issue by imontoyah** [time-series-foundation-models/lag-llama#97]
Hello Lag Llama team,

I would like to know if Lag Llama models take dates into account when fine-tuning the model and making predictions. 
To clarify, I am asking this because I need to understand the correct way to split my data for the forecasting process.

Thank you in advance for your assistance.

**Comment by ashok-arjun** [comment]
Yes, it takes dates into account. The model uses time features which are calculated from the dates.
